upload/inc/languages/english/admin/isango.lang.php

```diff
@@ -29,3 +29,6 @@
 $l['isango_input_mode_max'] = 'Big size buttons with text';
 $l['isango_input_mode_min'] = 'Small buttons with icon only';
 $l['isango_input_mode_pop'] = 'A select dropdown';
+
+$l['isango_uninstall'] = 'Isango - uninstallation';
+$l['isango_uninstall_message'] = 'Do you wish to drop ALL plugin data from the database? Selecting \"No\" will leave untouched:<ul><li>Linked accounts for users.</li>\n\n</ul>\nSelecting \"No\" will <em>not</em>, however, prevent the removal of:<ul><li>Plugin settings (including any saved settings).</li>\n<li>The plugin\'s stylesheet, \"isango.css\" (including any changes you\'ve made to it), accessible for each theme via the ACP\'s Templates & Style -> <a href=\"index.php?module=style-themes\">Themes</a> module.</li>\n<li>The plugin\'s templates (including any changes you\'ve made to them), accessible  under each template set via the ACP\'s Templates & Style -> <a href=\"index.php?module=style-templates\">Templates</a> module.</li>\n</ul>';
```

Comment:
There are quotes and escape issues in this language string. It appears like this:
![image](https://user-images.githubusercontent.com/21265591/112428853-d3d18580-8d61-11eb-9270-2c48c6672f37.png)

Either use double quotes or just don't escape.